extensions/khr/GL_KHR_shader_subgroup.txt

```diff
@@ -450,7 +450,8 @@ Additions to Chapter 3 of the OpenGL Shading Language Specification
           if the local workgroup size is not a multiple of the subgroup size.
 
         In fragment shaders, helper invocations participate in subgroup
-        operations.
+        operations. Such as when additional fragment shader invocations are
+        created to fill out a quad partially covered by a primitive.
```

Comment:
"Such as" makes this sound like a sentence fragment. Is this edit really necessary, or is the other one enough by itself?

Reply:
It is a sentence fragment. That's bad. 

It's not strictly necessary, but the original sentence seemed a bit sparse on its own given that it didn't say what circumstances would necessitate helper invocation participation.

My preference would be to say: `helper invocations participate in subgroup operations when . . . ` but I'm not sure I know that complete list of "whens", so I thought I'd add the one I was confident of. Perhaps the whens take too long to explain?

Reply:
We've had some internal discussions about questions like "when are helper invocations guaranteed to exist", "can they dynamically come in and out of existence", "is there a way to force them to exist at a given point in a program", things like that, and they don't have easy answers. We haven't gotten far enough to be able to state anything definitive about it, so I think we should continue to err on the side of being vague.

Reply:
Fair enough. Unfortunate that clarity is not possible, but better to be vague than misleading.